contracts/StakerV2.sol

```diff
@@ -940,9 +1026,12 @@ contract StakerV2 is IERC721Receiver, IERC1155Receiver, Ownable, ReentrancyGuard
       uint256[] calldata ids,
       uint256[] calldata values,
       bytes calldata data
-  ) external override returns (bytes4){
+  ) external override returns (bytes4) {
+
       return this.onERC1155BatchReceived.selector;
   }
 
-  function supportsInterface(bytes4 interfaceId) external view override returns (bool){}
+  function supportsInterface(bytes4 interfaceId) external view override returns (bool) {
```

Comment:
Where did we ever land with inheriting from `1155Holder` instead of `Receiver` to do away with this?